tech.json

```diff
@@ -1056,13 +1056,32 @@
         {
           "name": "canShinespark",
           "requires": [ "SpeedBooster" ],
-          "note": "The ability to use the Speed Booster to store a shinecharge, then fly in a direction until an obstacle is hit or Samus gets down to 29 energy.",
+          "note": [
+            "The ability to use the Speed Booster to store a shinecharge, then fly in a direction until an obstacle is hit or Samus gets down to 29 energy.",
+            "Assumes Samus can shinespark vertically or diagonally from the ground."
+          ],
           "extensionTechs": [
+            {
+              "name": "canHorizontalShinespark",
+              "requires": [ "canShinespark" ],
+              "note": [
+                "The ability to shinespark horizontally from the ground.",
+                "This can be achieved by starting without horizontal momentum and pressing jump and then forward while still holding jump (or repressing jump).",
+                "These inputs can be done in rapid succession or with a short delay.",
+                "Shinesparking in water can be done the same way, but requires a slightly longer delay between the jump and forward presses, and it is particularly important to ensure Samus has no horizontal momentum."
+              ]
+            },
             {
               "name": "canMidairShinespark",
               "requires": [ "canShinespark" ],
```

Comment:
midairShinespark works better as an extension of horizontal.

Reply:
Oh whoops. That's what I intended it to be.